src/index.ts

```diff
@@ -111,6 +115,9 @@ export function createSelectorCreator<
 
     const dependencies = getDependencies(funcs)
 
+    // do we want to try and use the equality fn from options? how do we account for other memoizers?
+    const cacheKeyComparator = createCacheKeyComparator(defaultEqualityCheck)
```

Comment:
Straight shallow equality is probably simplest, although I _can_ see an argument for using the specific memoization setup they provided.